Classes/Private/Manager/Audio/OCTAudioEngine.m

```diff
@@ -97,9 +97,23 @@ - (instancetype)init
 
     _AUGraphNodeInfo(_processingGraph, _ioNode, NULL, &_ioUnit);
 
+    _enableMicrophone = YES;
+
+    if (! ([self enableInputScope:error] &&
+           [self registerInputCallBack:error] &&
+           [self registerOutputCallBack:error] &&
+           [self initializeGraph:error])) {
+        return nil;
+    }
+
     return self;
 }
 
+- (instancetype)init
+{
+    return [[OCTAudioEngine alloc] init:nil];
```

Comment:
You don't need this method due my comment above.

Anyway, even if you'd needed it - it is wrong. You don't want to `alloc` engine once again. You just want to call `init` method from other `init`, simple as that
```
return [self init:nil];
```